keyboards/sofle/keymaps/default/config.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+/* The way how "handedness" is decided (which half is which),
+see https://docs.qmk.fm/#/feature_split_keyboard?id=setting-handedness
+for more options.
+*/
+#define MASTER_LEFT
+
+#define TAPPING_FORCE_HOLD
```

Comment:
I'm not a fan of this being in the default keymap.  But if you have no tap toggles, you should be okay.

Reply:
Removed